HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -1508,6 +1513,13 @@ def self.apply_hvac(runner, hpxml, hpxml_bldg, weather, convert_shared_systems)
       heat_pump.backup_heating_lockout_temp_isdefaulted = true
     end
 
+    # Default advanced defrost
+    hpxml_bldg.heat_pumps.each do |heat_pump|
+      next unless [HPXML::HVACTypeHeatPumpAirToAir, HPXML::HVACTypeHeatPumpMiniSplit, HPXML::HVACTypeHeatPumpRoom, HPXML::HVACTypeHeatPumpPTHP].include? heat_pump.heat_pump_type
+
+      heat_pump.backup_heating_lockout_temp_isdefaulted = true
+    end
```

Comment:
Huh? I don't understand this at all.

Reply:
Oops must be something didn't resolve correctly when refactoring the inputs. Will fix it.

Reply:
Oops must be something didn't resolve correctly when refactoring the inputs. Will fix it.